koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -743,7 +744,8 @@ def createContainer(self, src=None, target_info=None, arches=None,
             self.logger.debug("Starting %s with params: '%s'",
                               create_method, create_build_args)
             build_response = create_method(**create_build_args)
-
+        except OsbsValidationException as exc:
```

Comment:
Why not `OsbsException` as we agreed in comments?

Reply:
@rcerven suggested, and I agreed, that trapping OsbsException would eat *all* tracebacks, and we don't want to do that.